src/main/kotlin/io/github/mojira/arisa/modules/commands/FixCapitalizationCommand.kt

```diff
@@ -9,13 +9,23 @@ class FixCapitalizationCommand : Command1<String> {
                 """(?<=\.\s|^|!\s|\?\s|\n)[A-Z][A-Za-z-'0-9]*((\s|,\s|;\s|:\s)[A-Z][A-Za-z-'0-9]*)*(?=\.|${'$'}|!|\?|\n)"""
                     .toRegex()
 
+        val exceptions = listOf(
+            ' i ',
+            ' minecraft ',
+            ' mojang '
```

Comment:
The regex `\b` can be used to match word boundaries; I'm not sure how that works for `replace` though.

Reply:
replace takes a string iirc

Reply:
`Regex` seems to have its own [`replace` method](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/-regex/replace.html) :eyes: